apps/api/v1/views.py

```diff
@@ -0,0 +1,29 @@
+from django.db.models import Q
+
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework import filters, viewsets
+
+from resources.models import Resource
+
+from .serializers import ResourceSerializer
+
+
+class ResourceViewSet(viewsets.ReadOnlyModelViewSet):
+    queryset = Resource.objects.approved()
```

Comment:
We probably need some select_related here - things like names, organisations etc will be 1 extra query per row.